ItemPool.py

```diff
@@ -488,19 +488,19 @@
     'Stick Pot': 'Deku Stick Drop',
     'Nut Pot': 'Deku Nut Drop',
     'Nut Crate': 'Deku Nut Drop',
-    'Blue Fire': 'Blue Fire',
-    'Lone Fish': 'Fish',
-    'Fish Group': 'Fish',
-    'Bug Rock': 'Bugs',
-    'Bug Shrub': 'Bugs',
-    'Wandering Bugs': 'Bugs',
-    'Fairy Pot': 'Fairy',
-    'Free Fairies': 'Fairy',
-    'Wall Fairy': 'Fairy',
-    'Butterfly Fairy': 'Fairy',
-    'Gossip Stone Fairy': 'Fairy',
-    'Bean Plant Fairy': 'Fairy',
-    'Fairy Pond': 'Fairy',
+    'Blue Fire': 'Blue Fire Drop',
+    'Lone Fish': 'Fish Drop',
+    'Fish Group': 'Fish Drop',
+    'Bug Rock': 'Bugs Drop',
+    'Bug Shrub': 'Bugs Drop',
+    'Wandering Bugs': 'Bugs Drop',
+    'Fairy Pot': 'Fairy Drop',
+    'Free Fairies': 'Fairy Drop',
+    'Wall Fairy': 'Fairy Drop',
+    'Butterfly Fairy': 'Fairy Drop',
+    'Gossip Stone Fairy': 'Fairy Drop',
+    'Bean Plant Fairy': 'Fairy Drop',
+    'Fairy Pond': 'Fairy Drop',
```

Comment:
This looks incorrect since these are still called "Bugs" and "Fairy", etc.

Reply:
The actual get-items were added to the item list under those names and the drops were renamed, so this is correct. Though it might also be possible to remove the "drop" type of item and just use the get-items.